services/web/server/src/simcore_service_webserver/groups/_groups_repository.py

```diff
@@ -108,7 +108,7 @@ async def _get_group_and_access_rights_or_raise(
     *,
     caller_id: UserID,
     group_id: GroupID,
-    permission: Literal["read", "write", "delete"] | None,
+    check_permission: Literal["read", "write", "delete"] | None,
 ) -> Row:
     result = await conn.execute(
```

Comment:
since it raises, should the docstring not be explaining what it raises? and actually I always expect things to raise but that is a different topic.